packages/tokens/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@fluentui/tokens",
+  "version": "1.0.0-alpha.0",
```

Comment:
since this is not version 9.x, it will not be released automatically as a part of the v9 release with `react-components`.

I would suggest a follow up PR to update https://github.com/microsoft/fluentui/blob/905b3b2e74e472bd2d9cb825c0dc12d3dee4aa46/scripts/beachball/utils.ts#L47

and 

https://github.com/microsoft/fluentui/blob/905b3b2e74e472bd2d9cb825c0dc12d3dee4aa46/scripts/monorepo/isConvergedPackage.js#L19

so that we can have a better way of creating release scopes for what we consider 'v9'

Reply:
I would suggest using `tags` that we have in https://github.com/microsoft/fluentui/blob/master/workspace.json if we want to align with our gradual migration to NX (cc @Hotell) or finding some other way to define a v9 scope that does not depend on version number

Reply:
ok,
1. let's create a separate PR for the `@fluentui/tokens` stub.
2. make sure the release works for that package
3. merge this PR